src/sentry/integrations/vercel/integration.py

```diff
@@ -277,26 +279,14 @@ def update_organization_config(self, data):
             for name, val in zip(secret_names, values):
                 secrets.append(vercel_client.create_secret(vercel_project_id, name, val))
 
-            secrets.append("")
             for secret, env_var in zip(secrets, env_var_names):
                 self.create_env_var(vercel_client, vercel_project_id, env_var, secret)
 
         config.update(data)
         self.org_integration.update(config=config)
 
-    def env_var_already_exists(self, client, vercel_project_id, name):
-        return any(
-            [
-                env_var
-                for env_var in client.get_env_vars(vercel_project_id)
-                if env_var["key"] == name
-            ]
-        )
-
     def create_env_var(self, client, vercel_project_id, key, value):
-        if not self.env_var_already_exists(client, vercel_project_id, key):
```

Comment:
@ceorourke I would put the `try/except` logic here instead of in the `client. create_env_variable` function. We should keep the client functions to be simple wrappers for making requests (one client function only hits one API).